third_party/blink/renderer/core/farbling/brave_session_cache.cc

```diff
@@ -84,10 +85,10 @@ blink::WebContentSettingsClient* GetContentSettingsClientFor(
   blink::WebContentSettingsClient* settings = nullptr;
   if (!context)
     return settings;
-  // Avoid blocking fingerprinting in WebUI pages.
+  // Avoid blocking fingerprinting in WebUI, extensions, etc.
   const String protocol = context->GetSecurityOrigin()->Protocol();
-  if (protocol == url::kAboutScheme || protocol == "chrome" ||
-      protocol == "brave") {
+  if (protocol == url::kAboutScheme || protocol == "chrome-extension" ||
```

Comment:
do we have tests for this?

Reply:
if not, please add some

Reply:
Pushed as https://github.com/brave/brave-core/pull/15931/commits/7da2cec4c2f55ff7ec830c383e8bfcdf8d0a9994